docs-source/spring/content/development/cli/_index.md

```diff
@@ -73,82 +77,209 @@ The following is an example development workflow using the CLI.
 First, a connection is made to the Oracle server-side Spring Admin
 
 ```cmd
-oracle-spring:>connect
-URL (defaults to http://localhost:8080): 
-using default value... 
-Username (defaults to admin): 
-using default value... 
-Password (defaults to oraclespring): 
+obaas:>help connect
+NAME
+       connect - connect to the Oracle Spring admin console
+
+SYNOPSIS
+       connect --url String --username String
+
+OPTIONS
+       --url String
+       Spring Cloud admin server URL
+       [Optional, default = http://localhost:8080]
+
+       --username String
+       Spring Cloud username (will be prompted for password)
+       [Optional, default = admin]
+
+
+
+obaas:>connect
+Spring Cloud password (defaults to oraclespring): 
 using default value... 
-connect successful server version:121522 client version:121522
+connect successful server version:121522
 ```
 
 Then, an application namespace is created withe the `create` command. This namespace will contain the microservices that are deployed later.
 
 ```cmd
-oracle-spring:>create
-appName (defaults to cloudbank): 
-using default value... 
+obaas:>help create
+NAME
+       create - create an application/namespace
+
+SYNOPSIS
+       create --appName String
+
+OPTIONS
+       --appName String
+       application/namespace
+       [Optional, default = cloudbank]
```

Comment:
shouldn't the default be "application", since that is the pre-created and configured ns?